docs/src/components/CodeSnippet.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import Editor, { EDITOR_BACKGROUND_COLOR } from './Editor'
+
+export interface ICodeSnippetProps {
```

Comment:
here is a point for further thought: to me it seems a bit unreasonable to use 'I' prefix for interface types - this leads to the fiolloiwng situation where `interface` types and alias `type` types - both with exactly the same semantics, - are mixed:

```jsx
import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
```

Here we are importing two type interfaces (in general sense) for styles and variables - but for some reason one uses `I` prefix :) What I would suggest is to omit `I` prefix altogether for all types - this will make them consistent. Actually, the same approach React has taken for its types - quite a few of them are TS interfaces, but all the exposed names are consistent to the client as there are no `I` prefixes

Reply:
I have no opinion here other than it should follow the best practice.  It is nice to know when you can use `extends` (interfaces but not types).  However, it appears the [official recommendation](https://github.com/Microsoft/TypeScript-Handbook/issues/121) is to _not_ use prefixes.  Will update.

Reply:
I went to update this and it appears we have 44 interfaces in 24 files prefixed with "I" and only 4 interfaces in 3 files that aren't.  I think we should make a separate PR that addresses this and updates the team about the new standard.

I will still update my usage here to not include the prefix.